.github/workflows/update-standard.yml

```diff
@@ -0,0 +1,31 @@
+name: update-standard
+
+on:
+  push:
+    paths:
+    - 'dicom_standard/*.py'
+  schedule:
+    - cron: '0 0 1 * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.7'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Run Makefile
+        run: |
+          make clean -C dicom_standard
+          make updatestandard -C dicom_standard
+          make -C dicom_standard
+      - name: Run tests
+        run: pytest tests
```

Comment:
This may be a dumb question, but will this "step" still have access to the files run by the previous step that runs "make"?

Reply:
I believe so:
>Each step in a job executes on the same runner, allowing the actions in that job to share information using the filesystem.

[Source](https://help.github.com/en/actions/getting-started-with-github-actions/core-concepts-for-github-actions#step)

Reply:
Regarding the comment [here](https://github.com/innolitics/dicom-standard/pull/27#discussion_r391916402):
>Also, I suspect you were planning on doing this, but once we have generated the new files, maybe we could:
>
>1. detect if anything changes in the files
>2. if something did change, we can create new pypi releases automatically

We could create a second job that executes after the first, and pass it the data from the first: https://help.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts